### PR TITLE
Improve cosnsitency of git publish actions examples

### PR DESCRIPTION
**Pull request by BillyMichael**
## Hey, I just made a Pull Request!

Updates examples for all publish actions to ensure they are consistent with each other

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: plugins/scaffolder-backend-module-bitbucket/src/actions/bitbucket.examples.ts
Comment (awanlin) [inline plugins/scaffolder-backend-module-bitbucket/src/actions/bitbucket.examples.ts:23]:
```suggestion
      'Initializes a git repository with the content in the workspace, and publishes it to Bitbucket with the default configuration.',
```

File: plugins/scaffolder-backend-module-bitbucket/src/actions/bitbucket.examples.ts
Comment (awanlin) [inline plugins/scaffolder-backend-module-bitbucket/src/actions/bitbucket.examples.ts:75]:
```suggestion
      'Initializes a Bitbucket repository with a default branch, if not set defaults to main',
```

I can't get clear confirmation on this in the docs but generally it seems the default was changed to main at some point after 2020. I'm fine with being wrong on this, I always have a hard time finding things in the Atlassian docs.

File: plugins/scaffolder-backend-module-gitea/src/actions/gitea.examples.ts
Comment (awanlin) [inline plugins/scaffolder-backend-module-gitea/src/actions/gitea.examples.ts:23]:
```suggestion
      'Initializes a git repository with the content in the workspace, and publishes it to Gitea with the default configuration.',
```

File: plugins/scaffolder-backend-module-github/src/actions/github.examples.ts
Comment (awanlin) [inline plugins/scaffolder-backend-module-github/src/actions/github.examples.ts:23]:
```suggestion
      'Initializes a git repository with the content in the workspace, and publishes it to Github with the default configuration.',
```

File: plugins/scaffolder-backend-module-gitlab/src/actions/gitlab.examples.ts
Comment (awanlin) [inline plugins/scaffolder-backend-module-gitlab/src/actions/gitlab.examples.ts:23]:
```suggestion
      'Initializes a git repository with the content in the workspace, and publishes it to Gitlab with the default configuration.',
```

**Review by awanlin — CHANGES_REQUESTED** [review]
Thanks for this contribution @BillyMichael! 🚀 

Left some comments, mostly nits. There was some wording that seemed hard to read from my perspective "of" vs "with". Also, would be good to clarify the branch default of `master` vs `main` in a couple of cases.

File: plugins/scaffolder-backend-module-azure/src/actions/azure.examples.ts
Comment (BillyMichael) [reply plugins/scaffolder-backend-module-azure/src/actions/azure.examples.ts:57]: Yes, the action sets the defaultBranch to master by default 

```const {
        repoUrl,
        defaultBranch = 'master',
        gitCommitMessage = 'initial commit',
        gitAuthorName,
        gitAuthorEmail,
      } = ctx.input;```

File: plugins/scaffolder-backend-module-bitbucket/src/actions/bitbucket.examples.ts
Comment (BillyMichael) [reply plugins/scaffolder-backend-module-bitbucket/src/actions/bitbucket.examples.ts:75]: Same as my previous comment. DefaultBranch is set to master in the action

```
const {
        repoUrl,
        description,
        defaultBranch = 'master',
        repoVisibility = 'private',
        enableLFS = false,
        gitCommitMessage = 'initial commit',
        gitAuthorName,
        gitAuthorEmail,
      } = ctx.input;
```

File: plugins/scaffolder-backend-module-bitbucket/src/actions/bitbucket.examples.test.ts
Comment (BillyMichael) [reply plugins/scaffolder-backend-module-bitbucket/src/actions/bitbucket.examples.test.ts:274]: I've just looked through the code and this would involve changes to tests that sit outside the scope of "examples". Ill open another PR to remove `tokenlols`

**Comment by BillyMichael** [comment]
> Thanks for this contribution @BillyMichael! 🚀
> 
> Left some comments, mostly nits. There was some wording that seemed hard to read from my perspective "of" vs "with". Also, would be good to clarify the branch default of `master` vs `main` in a couple of cases.

Thank you. Appreciate the feedback. I've made the changes apart from the token as that falls outside the scope of what i was trying to achieve here. Ill open another PR for the token changes.

File: plugins/scaffolder-backend-module-bitbucket/src/actions/bitbucket.examples.test.ts
Comment (awanlin) [reply plugins/scaffolder-backend-module-bitbucket/src/actions/bitbucket.examples.test.ts:274]: Ok, that works for me, thanks for being open to some "extra" work 👍

**Review by awanlin — APPROVED** [review]
Awesome, this looks great to me now, thanks @BillyMichael! 🚀 

We'll need a review from the maintainers now.

File: plugins/scaffolder-backend-module-bitbucket/src/actions/bitbucket.examples.test.ts
Comment (tudi2d) [inline plugins/scaffolder-backend-module-bitbucket/src/actions/bitbucket.examples.test.ts:192]: I would strongly vouche to keep `main` where possbile. We are only historically stuck with `master` on this repository. In new repositories we are also using `main` over `master`.

File: plugins/scaffolder-backend-module-github/src/actions/github.examples.ts
Comment (tudi2d) [inline plugins/scaffolder-backend-module-github/src/actions/github.examples.ts:23]: Let's try to use `GitHub` & `GitLab` over `Github` & `Gitlab` in this PR :)

```suggestion
      'Initializes a git repository with the content in the workspace, and publishes it to GitHub with the default configuration.',
```

**Review by tudi2d — COMMENTED** [review]
thanks for your suggestions! 2 small comments.

File: plugins/scaffolder-backend-module-bitbucket/src/actions/bitbucket.examples.test.ts
Comment (BillyMichael) [reply plugins/scaffolder-backend-module-bitbucket/src/actions/bitbucket.examples.test.ts:192]: This is a consequence of modifying the examples to not longer specify a custom branch for **every** example. By defauly the bitbucket scaffolder module sets the default branch to master if one if not specified.

File: plugins/scaffolder-backend-module-bitbucket/src/actions/bitbucket.examples.ts
Comment (BillyMichael) [inline plugins/scaffolder-backend-module-bitbucket/src/actions/bitbucket.examples.ts:105]: @tudi2d it's the removal of these lines from the examples which results in the default branch being used, which is master.

**Review by freben — APPROVED** [review]
OK since this changes only examples, and it was up to the module itself to decide what the default branch is, this should be ok to ship

**Pull request merged**
